vcsclient/bitbucketcloud.go

```diff
@@ -420,6 +443,35 @@ func extractCommitDetailsFromResponse(commit interface{}) (commitDetails, error)
 	return res, nil
 }
 
+func extractCommentsFromResponse(comments interface{}) (*commentsResponse, error) {
+	var res commentsResponse
```

Comment:
Let's move this line a few lines down, as close as possible to the code that needs it.